packages/schematics/src/utils/rules/format-files.ts

```diff
@@ -28,15 +11,53 @@ export function formatFiles(
     return noop();
   }
   return (host: Tree, context: SchematicContext) => {
-    const packageJson = readJsonInTree(host, 'package.json');
-    if (packageJson.scripts && packageJson.scripts.format) {
-      context.addTask(new FormatFiles());
-    } else {
-      context.logger.warn(stripIndents`
-        Files were not formated during this code generation.
-        The "format" npm script is missing in your package.json.
-        Please either add a format script or pass --skip-format.
-      `);
+    const files = new Set(
+      host.actions
+        .filter(action => action.kind !== 'd' && action.kind !== 'r')
+        .map(action => {
+          switch (action.kind) {
+            case 'o':
+              return {
+                path: action.path,
+                content: action.content.toString()
+              };
+            case 'c':
+              return {
+                path: action.path,
+                content: action.content.toString()
+              };
+          }
+        })
+    );
+    if (files.size === 0) {
+      return host;
     }
+    return from(files).pipe(
+      filter(file => host.exists(file.path)),
+      mergeMap(async file => {
+        const systemPath = appRoot.resolve(file.path);
```

Comment:
do we have to use `resolve` or can we use simple path construction? We don't use resolve anywhere else.